src/cljc/markdown/transformers.cljc

```diff
@@ -125,9 +125,9 @@
     (str " " text) text))
 
 (defn paragraph
-  [text {:keys [eof heading inline-heading temp hr code lists blockquote paragraph last-line-empty?] :as state}]
+  [text {:keys [eof heading inline-heading temp hr code codeblock lists blockquote paragraph last-line-empty? end-codeblock?] :as state}]
   (cond
-    (and paragraph lists)
+    (or (and paragraph lists) (and paragraph codeblock))
```

Comment:
Using the backticks you'll still get content that's broken up:

    "hey\n\n```\nhey\n```\n\nhey"

    <p>hey</p><pre><code>hey
    </code></pre><p>hey</p>